lib/registries/addon/drafts/draft/-components/managers/license-picker-manager/component.ts

```diff
@@ -0,0 +1,87 @@
+import { tagName } from '@ember-decorators/component';
+import Component from '@ember/component';
+import { action } from '@ember/object';
+import { alias, not, sort } from '@ember/object/computed';
+import { inject as service } from '@ember/service';
+import { timeout } from 'ember-concurrency';
+import { task } from 'ember-concurrency-decorators';
+import DS from 'ember-data';
+
+import { ChangesetDef } from 'ember-changeset/types';
+import { layout } from 'ember-osf-web/decorators/component';
+import DraftRegistration from 'ember-osf-web/models/draft-registration';
+import License from 'ember-osf-web/models/license';
+import { NodeLicense } from 'ember-osf-web/models/node';
+import { QueryHasManyResult } from 'ember-osf-web/models/osf-model';
+
+import template from './template';
+
+@tagName('')
+@layout(template)
+export default class LicensePickerManager extends Component {
+    @service store!: DS.Store;
+
+    // required
+    changeset!: ChangesetDef;
+    model!: DraftRegistration;
+
+    // private
+    licensesAcceptable!: QueryHasManyResult<License>;
+    currentLicense!: License;
+    currentNodeLicense!: NodeLicense;
+
+    @alias('model.license') selectedLicense!: License;
+    @not('currentLicense') fieldIsEmpty!: License;
```

Comment:
```suggestion
    @not('selectedLicense') fieldIsEmpty!: License;
```

Reply:
Verify whether the `fieldIsEmpty` field is necessary. Remove if not.